pages/common/bfg.md

```diff
@@ -0,0 +1,16 @@
+# bfg
+
+> Remove large files or passwords from Git history like git-filter-branch
+> More information: <https://rtyley.github.io/bfg-repo-cleaner/>.
+
+- Remove a file with sensitive data and leave the latest commit untouched:
+
+`bfg --delete-files {{FILE-WITH-SENSITIVE-DATA}}`
+
+- Replace all text listed in passwords.txt wherever it can be found in the repository's history:
+
+`bfg --replace-text {{passwords.txt}}`
```

Comment:
```suggestion
- Remove all text mentioned in text.txt wherever it can be found in the repository's history:

`bfg --replace-text {{text.txt}}`
```

Reply:
I'd rather something a bit more generic, e.g.
```suggestion
- Remove all text mentioned in the specified file wherever it can be found in the repository's history:

`bfg --replace-text {{path/to/file.txt}}`
```